internal/api/stages/fakegithub/events.go

```diff
@@ -3,14 +3,15 @@ package fakegithub
 import (
 	"bytes"
 	"fmt"
+	"testing"
+
 	"github.com/form3tech-oss/github-team-approver-commons/pkg/configuration"
 	"github.com/google/go-github/v42/github"
 	"github.com/stretchr/testify/require"
-	"testing"
 )
 
-// ReviewEvent Capturing the data we require in sending an event
-type ReviewEvent struct {
+// Event Capturing the data we require in sending an event
+type Event struct {
```

Comment:
Since both `PullRequestEvent` and `PullRequestReviewEvent` share the same data attributes, I renamed the type to `Event` and updated it with two specific constructors.